config.js

```diff
@@ -1,24 +1,63 @@
+import { ClientSecretCredential } from "@azure/identity";
+import { TokenCredentialAuthenticationProvider } from "@microsoft/microsoft-graph-client/authProviders/azureTokenCredentials";
+
 const MAILBOX_URI = process.env.MAILBOX_URI;
 const EMAIL_PROTOCOL = process.env.EMAIL_PROTOCOL || 'smtp';
 const FROM_NAME = process.env.FROM_NAME || '';
 const HOURS_DELIVERING_TIMEOUT = process.env.HOURS_DELIVERING_TIMEOUT || 1;
 const HOURS_SENDING_TIMEOUT = process.env.HOURS_SENDING_TIMEOUT || .5;
 const MAX_BATCH_SIZE = process.env.MAX_BATCH_SIZE || 10;
+const MAX_BATCH_WAIT_TIME = process.env.MAX_BATCH_WAIT_TIME || 1000;
 const MAX_RETRY_ATTEMPTS = process.env.MAX_RETRY_ATTEMPTS || 5;
 const WELL_KNOWN_SERVICE = (process.env.WELL_KNOWN_SERVICE || '').toLowerCase();
-const CRON_FREQUENCY = process.env.EMAIL_CRON_PATTERN || '* * 1 * * *';
+const CRON_FREQUENCY = process.env.EMAIL_CRON_PATTERN || '0 */1 * * *';
```

Comment:
Ah, I didn't realize that the library used a different cron pattern. Will revert the change and add something in the readme.

But does this mean that the cron pattern gets executed each second of each minute of every first hour of a day?

Reply:
> But does this mean that the cron pattern gets executed each second of each minute of every first hour of a day?

Yes, silly, but since that was the default before I would leave it untouched for now.

Reply:
The description of the env var in the README still states `Default: once every hour`